Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git", :revision => "10a9bee148d2a833069d6cc27aea56af3092dc6e"
+  version "0.7.2"
+
+  depends_on "rust" => :build
+
+  conflicts_with "hadoop", :because => "both install `yarn` binaries"
```

Comment:
hadoop installs yarn?

Reply:
According to the formula, [yes it does](https://github.com/Homebrew/homebrew-core/blob/d1169efbb553628d2d64129b4345084466b956eb/Formula/hadoop.rb#L13)

Reply:
> hadoop installs yarn?

It's another yarn, not node package manager 🙂